index.js

```diff
@@ -85,6 +89,11 @@ class Markdown extends Component {
 
         const { styles } = this.state;
 
+        if (this.props.renderList) {
+            const children = node.props.children.map((node) => this.renderNodes(node, key, { ordered }));
+            return this.props.renderList(ordered, children);
+        }
```

Comment:
Children should already be rendered by the other functions, we just pass them to a List renderer which should build the container and interpolate the children?